libs/multicall/src/hooks/useMultipleContractSingleData.ts

```diff
@@ -42,22 +42,23 @@ export function useMultipleContractSingleData<T = Result>(
   return useSWR<(T | undefined)[] | null>(
     !calls?.length || !provider
       ? null
-      : [provider, calls, multicallOptions, methodName, contractInterface, chainId, 'useMultipleContractSingleData'],
+      : [
+          provider,
+          calls,
+          multicallOptions,
+          methodName,
+          contractInterface,
+          chainId,
+          calls.length,
```

Comment:
Just in case added `calls.length` to the swr key, because serialized `calls` doesn't give any change.